agent/config/config_windows.go

```diff
@@ -19,6 +19,11 @@ import (
 	"path/filepath"
 	"time"
 
+	"github.com/cihub/seelog"
```

Comment:
nit: can you restructure the imports to follow the golang package grouping

Reply:
can you elaborate? i've already run ```go fmt```/```go imports```

Reply:
```
import (
    stdlib

    current_project

    company

    all others 
)
```

Reply:
The static checks are expecting the structure of the inputs to be the result of running `go imports`. Where is that order coming from?

Reply:
not sure i follow. is the suggestion to do something like:
```
import (
	"os"
	"path/filepath"
	"time"

	"golang.org/x/sys/windows"

	"github.com/aws/amazon-ecs-agent/agent/dockerclient"
	"github.com/aws/amazon-ecs-agent/agent/utils"

	"github.com/cihub/seelog"
	"github.com/hectane/go-acl/api"
)
```

Reply:
Yep, I prefer this.

Reply:
@angelcar you okay with this too? ran ```go imports``` after and it seems to accept this order.

Reply:
If the static checks are ok with it, I'm ok with it =)

Reply:
done